### ISSUE TITLE
Add alternative text support

### ISSUE THREAD

**Issue by Taha-Hassan-Git** [tldraw/tldraw#4389]
### What's the feature?

# Alternative text for non-text shapes. 

Images being the most obvious, videos as well. But also geoshapes. At the moment a screen reader will just announce that there is an image, and won't even read the text label. What if you got something more descriptive like: "Black rectangle, text: I'm a rectangle"?

Miro, Canva and Figma/Figjam all have alt text for images, and they can be edited via a context menu. Canva and Miro also offer an AI suggestion for the image description, which is super helpful. However, Canva don't have good descriptions for non-image and non-text shapes, i.e sticky notes or triangles etc, they are simply described as images.

## UI

What existing implementations of alt text editing features look like:

### Figma

The most stripped-down and austere looking UI. You get an idea of what it's for and then a tiny text box to write in, implying your description shouldn't be too verbose. Could do with being bigger and a bit friendlier looking imo.

<img width="471" alt="Screenshot 2024-08-15 at 11 39 04" src="https://github.com/user-attachments/assets/bbf95864-cda2-48d4-b84a-cfc3252c88a3">

### Canva

The most verbose UI solution. It surfaces more information and guidance for writing alt-text along with an explicit character limit. Big, chunky AI suggest for me button. Mark as decorative is a nice feature, though not necessarily relevant to us- I think this is for generating accessible PDFs of designs.

<img width="471" alt="Screenshot 2024-08-15 at 11 39 04" src="https://github.com/user-attachments/assets/a84b7f4e-8641-48bd-adfb-04e3fc480d65">

### Miro

Something of a middle-ground in terms of verbosity. Doesn't use the phrase alternative text for some reason? The AI suggested description is surfaced before you click the button, which is interesting.

<img width="471" alt="Screenshot 2024-08-15 at 11 39 04" src="https://github.com/user-attachments/assets/53462303-ffdc-491c-b857-71b657890ba5">

### Where would you like this feature?

The tldraw.com website

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct
